### DISCUSSION TITLE
Brew cask without brew?

### DISCUSSION THREAD

**Discussion by hedefalk** [Homebrew/discussions#418] [Casks]
I'm moving to nix for all things cli, but I still want to rely on homebrew for the casks. My impression is that the casks are actually a lot simpler - is there any way to use brew cask without brew and it's dependency on xcode devtools?

**Comment by SMillerDev** [comment]
That would just be downloading installers from upstream's website. That's all casks do.

**Comment by carlocab** [comment]
Try https://docs.brew.sh/Installation#untar-anywhere. That will bypass the CLT installation. If you use it exclusively for casks then I think it will stay that way, but I'm not sure.

**Reply by SMillerDev** [reply]
That's still installing brew though. The requirement is "without brew"
